src/api/organization-settings.ts

```diff
@@ -1,69 +1,55 @@
-import { gql } from "@apollo/client";
-
 import type { RequestAutoApproveType } from "./organization-membership";
 
 export interface OrganizationSettingsInput {
-  defaulTexterApprovalStatus: RequestAutoApproveType | null;
+  // Owner
   optOutMessage: string | null;
-  numbersApiKey: string | null;
-  trollbotWebhookUrl: string | null;
   showContactLastName: boolean | null;
   showContactCell: boolean | null;
   confirmationClickForScriptLinks: boolean | null;
+  defaulTexterApprovalStatus: RequestAutoApproveType | null;
```

Comment:
Only thing to add is that this should probably be corrected to `defaultTexterApprovalStatus` but the typo already exists so not introduced in this PR 🙂